script/package.ts

```diff
@@ -84,7 +84,7 @@ function packageWindows() {
   }
 
   const iconUrl =
-    'https://desktop.githubusercontent.com/github-desktop/app-icon.ico'
+    'https://raw.githubusercontent.com/desktop/desktop/development/app/static/logos/icon-logo.ico'
```

Comment:
There were a few options to pick from:
1) `app-icon.ico` (desktop.githubusercontent.com) can be updated to use the updated icon instead of using `icon-logo.ico` (raw.githubusercontent.com) which means that this pull request is no longer needed.
2) `iconUrl` variable can be replaced with `iconPath` with logic that is similar to `splashScreenPath` which means that there is no need to hardcode the URL.
3) `iconUrl` variable can be updated with the latest ICO file from `app/static/logos` using the development branch.